js/components/importer.js

```diff
@@ -0,0 +1,57 @@
+define(['knockout', 'text!./importer.html', 'atlas-state', 'databindings'], function (ko, view, sharedState) {
```

Comment:
Per our skype conversation, I believe this file is not needed and can be removed. Can you remove this? We should then be able to get this merged.